source/crud/crud.rst

```diff
@@ -1438,35 +1438,47 @@ Any result class with all parameters marked NOT REQUIRED is ultimately NOT REQUI
 
   }
 
-
+~~~~~~~~~~~~~~
 Error Handling
 ~~~~~~~~~~~~~~
 
-Defined below are error and exception types that should be reported from the various write methods. Since error types across languages would be impossible to reconcile, the below definitions represent the fields and names for the information that should be present. Structure isn't important as long as the information is available.
+Defined below are error and exception types that should be reported from the
+various write methods. Since error types across languages would be impossible to
+reconcile, the below definitions represent the fields and names for the
+information that should be present. Structure isn't important as long as the
+information is available.
 
-Drivers SHOULD report errors however they report other server errors: by raising an exception, returning "false", or another idiom that is consistent with other server errors.
+Drivers SHOULD report errors however they report other server errors: by raising
+an exception, returning "false" and populating an error struct, or another idiom
```

Comment:
"returning false" didn't make sense on its own, as there's way to actually report an error in that case. I assumed this was referring to libmongoc, so I'm clarifying that it returns false _and_ populates an error struct.